### PR TITLE
Add а field PhoneID

### PR DESCRIPTION
**Pull request by Albina-art**
Add а field PhoneID to the structure Informations. To be able to change this field

### CODE REVIEW

**Comment by ahmdrz** [comment]
Hi ,
Here 2 questions...
1. Why you pushing to `new` branch ?
2. Is it necessary to change PhoneID , Can you explain it ?

**Comment by Albina-art** [comment]
Hi,
1) Oops! I mixed it up.
2) To set a value PhoneID, if you have guid, login, deviceID, password and phoneID.
Then it would be nice to ask all this. 
Thanks for the quick response)

**Comment by Albina-art** [comment]
And do not generate again if you already have data( deviceID, phoneID,guid)

File: response/types.go [hunk 1]
```diff
@@ -217,40 +217,39 @@ type GetUserID struct {
...
-			Height int    `json:"height"`
-			Width  int    `json:"width"`
-			URL    string `json:"url"`
-		} `json:"hd_profile_pic_url_info"`
-		GeoMediaCount int    `json:"geo_media_count"`
-		IsBusiness    bool   `json:"is_business"`
-		Biography     string `json:"biography"`
-		HasChaining   bool   `json:"has_chaining"`
+		IsPrivate                  bool                  `json:"is_private"`
+		ExternalLynxURL            string                `json:"external_lynx_url"`
```
Comment (ahmdrz) [inline response/types.go]: Please use `gofmt`

**Comment by ahmdrz** [comment]
Nice , It will be good , But `phone_id` is only on [L151](https://github.com/ahmdrz/goinsta/blob/c7295817eb1d9deb0f6741e4fa52be3e1633017b/goinsta.go#L151)

Any way...
please make new pull request and push it on `master` branch and take a look at my last commit on your changes.

Special thank you for help us to improve this library.

Regards.

**Pull request closed without merging**